### ISSUE TITLE
env wrong

### ISSUE THREAD

**Issue by htharoldht** [pavanchhatpar/copynet-tf#1]
I've already set the DATA_DIR to '/root/data_copynet-tf', but when I run './setup-env.sh', it always shows as belows
'''sh
./setup-env.sh: 28: ./setup-env.sh: source: not found
.env should set DATA_DIR
'''
![图片](https://user-images.githubusercontent.com/31093493/82183151-075cab80-9918-11ea-9f79-9ab2c6b8ba68.png)
![图片](https://user-images.githubusercontent.com/31093493/82183247-3246ff80-9918-11ea-9795-216af5a4c5f3.png)

I want to know what's wrong?
I just want to run the demo "greeting"

**Comment by pavanchhatpar** [comment]
Hey, thanks for reporting this. This repository is quite new and I might have not done things in the best possible way.

I run the setup script inside a "bin/bash" terminal, while yours looks like a "/bin/sh" terminal. I've fixed this in the script no to work on any terminal [Commit link](https://github.com/pavanchhatpar/copynet-tf/commit/5eeef53cd28ae995d4cb8e4e190c190b227efdbf)

Also, in the screenshot I see "data -> ./data" which is incorrect as per your "DATA_DIR" value, probably give this a fresh start, "data" folder should point to "DATA_DIR" value and the script will assume that "DATA_DIR", which is "/root/data_copynet-tf", exists and you're able to write to it.

Don't forget to give --no-docker option if you don't want a docker image to be built.

**Comment by pavanchhatpar** [comment]
I've added demos to run it on Google Colab, please refer to those for an end-to-end understanding of the setup. Adding the links here for your reference,
 - [Running with the package](https://colab.research.google.com/drive/1fztwQ7baqrLhaKGnL4v_p_SjCP0yVESI?usp=sharing) (Recommended method)
 - [Running from source](https://colab.research.google.com/drive/1by8Ob-6JVN9_RSuDFEKAwJYOHIgPnN4f?usp=sharing)

At a later point I might clean up the code a bit, but for now, let me know if this helps you so I can close the issue

**Comment by htharoldht** [comment]
Thank you very much for your reply, I have already manually configured the environment with reference to the dockerfile.

I used to use vscode before, but I haven't used it as well as you are, so I haven't been clear about why the .env file is used, until I encounter a relative path guide package that will cause an operation error. I have no time to configure my vscode for the time being. I entered Pycharm, although the keymap of that one made me very headache

In addition, due to the existence of the national firewall, I cannot connect to Google Colab. In addition, thanks again.
